docs/developer-guide/api-reference.md

```diff
@@ -0,0 +1,1096 @@
+# zot API Command Reference
+
+> :point_right: This article describes the zot REST API commands, parameters, and responses.
+
+The information presented here is adapted from the interactive OpenAPI (formerly swagger) [JSON file](https://github.com/project-zot/zot/blob/main/swagger/swagger.json) in the zot Github project.
+
+For instructions and examples of how to use the zot API, see [Using the zot API](../developer-guide/api-user-guide.md).
+
+
+## /auth/apikey
```

Comment:
```suggestion
## /zot/auth/apikey
```

Please update the file everywhere we mention an /auth/ URL.

Reply:
Done